### DISCUSSION TITLE
Add connection point to object with simple key+mouse

### DISCUSSION THREAD

**Discussion by MastrUsr** [jgraph/drawio-desktop#1501] [Ideas]
It would be nice to easily add connection points without opening the edit dialog, which isn't very intuitive.. For example, holding ctrl and clicking with the mouse would add the point.

![image](https://github.com/jgraph/drawio-desktop/assets/34539988/34c8cd01-b4cf-4332-845c-930416de6cc5)

**Comment by nbuhr** [comment]
Quickly adding connection points, especially those that snap to grid lines when adding, would be immensely beneficial.

In the current system, connection points snap the _side_ of the X to the grid, rather than the center, and the grid isn't shown by default. It would be more intuitive to show the edits on the shape itself rather than a pop-up dialogue box.
